src/server/audit/schema.ts

```diff
@@ -51,6 +55,37 @@ export const AuditSchema = z.object({
 
     "update.public-status",
     "update.private-status",
+    "update.created",
+    "update.updated",
+    "update.cloned",
+    "update.shared",
+    "update.unshared",
+
+    "apiKey.created",
+    "apiKey.deleted",
+
+    "bucket.created",
+
+    "stripe.session-created",
+    "stripe.billingPortalSession-created",
+
+    "dataroom.created",
+    "dataroom.updated",
+    "dataroom.shared",
+    "dataroom.deleted",
+
+    "role.created",
+    "role.updated",
+    "role.deleted",
+
+    "template.field-created",
+    "template.created",
+
+    "passkey.created",
+    "passkey.authOptions-created",
+    "passkey.registrationOptions-created",
```

Comment:
Please change both `authOptions... and registration...` to `passkey.updated`, details will be on summary